source/collimation.f90

```diff
@@ -3333,39 +3259,41 @@ subroutine collimate_end_collimator(stracki)
 
 end subroutine collimate_end_collimator
 
-!>
-!! collimate_end_sample()
-!! This routine is called from trauthin after each sample
-!! has been tracked by thin6d
-!<
-subroutine collimate_end_sample(j)
+! ================================================================================================ !
+!  Collimate Exit
+! ================================================================================================ !
+subroutine collimate_exit
 
   use parpro
   use mod_common
-  use mod_common_main
   use mod_commons
-  use mod_common_track
   use mod_common_da
+  use mod_common_main
+  use mod_common_track
   use crcoall
   use coll_db
   use mod_units
+  use string_tools
 #ifdef ROOT
   use root_output
 #endif
 #ifdef HDF5
   use hdf5_output
   use hdf5_tracks2
 #endif
+#ifdef G4COLLIMATION
+  use geant4
+#endif
 
   implicit none
 
-  integer, intent(in) :: j
+  ! integer, intent(in) :: j
```

Comment:
I would completely kill the line

Reply:
Yeah, it's done in a later PR. There's a lot to clean up when we're done.